executor/executor.go

```diff
@@ -328,9 +330,43 @@ func (executor *Executor) execute(
 			}
 			return response, nil
 		}
-		msg := "decision executed successfully"
 
-		executor.logger.Infof(ctx, msg)
+		// short pooling to trigger pod status with max 15 minutes
+
+		statusMap := make(map[string]string)
+		statusMap["Running"] = "pod restarted successfully"
+		statusMap["Failed"] = "pod failed to restart"
+		statusMap["Unknown"] = "pod status is unknown"
+		statusMap["PodInitializing"] = "pod restarting"
+
+		backoff := 15 * time.Minute
+		msg := "pod restarting exceeded timout (15 min)"
+		start := time.Now()
+		timeout := int(backoff.Seconds())
```

Comment:
No need to do that. `time.Now() - start < backoff` should work

Reply:
Other than that it's fine, when you change that you can merge without waiting for me to review it again

Reply:
because the timeout variable not from type time

Reply:
You don't need the `timeout` variable at all. The `backoff` variable should be enough, like I used it in my example

Reply:
you are right but time.Now() is not a comparable unit it is of type time so you have to choose a unit to use this operation (-) and the most suitable one is seconds as we make short pooling every 15 second